### ISSUE TITLE
Type of parameters in methods

### ISSUE THREAD

**Issue by YevhenKap** [DinoLeung/TeleDart#36]
I have wrote this code:
```dart
import 'package:teledart/teledart.dart';
import 'package:teledart/telegram.dart';
import 'package:teledart/model.dart';

void main(List<String> args) {
  TeleDart t = TeleDart(Telegram('token'), Event());
  t.startFetching();

  final btns = <List<KeyboardButton>>[<KeyboardButton>[KeyboardButton(text: 'Yes'), KeyboardButton(text: 'No')]];

  final keyboard = ReplyKeyboardMarkup(keyboard: btns);

  t.onCommand('start').listen((msg) => t.replyMessage(msg, 'Some question?', reply_markup: keyboard));
}
```
And then a have got an error:

> Unhandled exception:
> HttpClientException:
> HttpClientException:
 > 400 Bad Request: can't parse keyboard button: Field "request_contact" must be of type Boolean
> #0      Telegram.sendMessage (package:teledart/src/telegram/telegram.dart:109:29)<asynchronous suspension>
> #1      TeleDart.replyMessage (package:teledart/src/teledart/teledart.dart:235:16)
> #2      main.<anonymous closure> (file:///Users/ky/Projects/medical_bot/bin//bot.dart:13:42)

When I trying fix this (set true or false to parameter) - a new similar error is coming with other parameter. But they are optional and if I don't give them boolean value, they shouldn't influence my request.

**Comment by DinoLeung** [comment]
Thanks for rising issue again, will look into this later tonight. Looks like model parsed incorrectly before sending to the server.

**Comment by DinoLeung** [comment]
@YevhenKap I have addressed the issue, which is occurred by incorrect configuration of [json_serializable](https://pub.dartlang.org/packages/json_serializable). will be fix in 0.021 release, which i'm going to push it up shortly 😃
